[owm] Error message (duplicate channel) each time the thing definition is changed

I have put the Bridge and Thing definition into a text file. Each time the file is changed, I get an error message about a duplicate channel.

Here is an excerpt of the debug output:
```
2018-12-26 21:25:45.793 [INFO ] [el.core.internal.ModelRepositoryImpl] - Refreshing model 'owm.things'
2018-12-26 21:25:46.010 [DEBUG] [nal.handler.OpenWeatherMapAPIHandler] - Dispose OpenWeatherMap API handler 'openweathermap:weather-api:api'.
2018-12-26 21:25:46.016 [DEBUG] [nal.handler.OpenWeatherMapAPIHandler] - Stop refresh job.
2018-12-26 21:25:46.021 [DEBUG] [nal.handler.OpenWeatherMapAPIHandler] - Initialize OpenWeatherMap API handler 'openweathermap:weather-api:api'.
2018-12-26 21:25:46.029 [DEBUG] [nal.handler.OpenWeatherMapAPIHandler] - Start refresh job at interval 30 min.
2018-12-26 21:25:46.044 [DEBUG] [nWeatherMapWeatherAndForecastHandler] - Initialize OpenWeatherMapWeatherAndForecastHandler handler 'openweathermap:weather-and-forecast:api:brilon'.
2018-12-26 21:25:46.052 [DEBUG] [nWeatherMapWeatherAndForecastHandler] - Rebuilding thing 'openweathermap:weather-and-forecast:api:brilon'.
2018-12-26 21:25:46.056 [DEBUG] [nWeatherMapWeatherAndForecastHandler] - Rebuilding daily forecast channel groups.
2018-12-26 21:25:46.061 [DEBUG] [andler.AbstractOpenWeatherMapHandler] - Building channel group 'forecastToday' for thing 'openweathermap:weather-and-forecast:api:brilon'.
2018-12-26 21:25:46.069 [DEBUG] [andler.AbstractOpenWeatherMapHandler] - Building channel group 'forecastTomorrow' for thing 'openweathermap:weather-and-forecast:api:brilon'.
2018-12-26 21:25:46.077 [DEBUG] [andler.AbstractOpenWeatherMapHandler] - Building channel group 'forecastDay2' for thing 'openweathermap:weather-and-forecast:api:brilon'.
2018-12-26 21:25:46.086 [DEBUG] [andler.AbstractOpenWeatherMapHandler] - Building channel group 'forecastDay3' for thing 'openweathermap:weather-and-forecast:api:brilon'.
2018-12-26 21:25:46.094 [DEBUG] [andler.AbstractOpenWeatherMapHandler] - Building channel group 'forecastDay4' for thing 'openweathermap:weather-and-forecast:api:brilon'.
2018-12-26 21:25:46.103 [DEBUG] [andler.AbstractOpenWeatherMapHandler] - Building channel group 'forecastDay5' for thing 'openweathermap:weather-and-forecast:api:brilon'.
2018-12-26 21:25:46.129 [ERROR] [nal.common.AbstractInvocationHandler] - An error occurred while calling method 'ThingHandler.thingUpdated()' on 'org.eclipse.smarthome.binding.openweathermap.internal.handler.OpenWeatherMapWeatherAndForecastHandler@16f199f': Duplicate channels openweathermap:weather-and-forecast:api:brilon:forecastToday#time-stamp
java.lang.IllegalArgumentException: Duplicate channels openweathermap:weather-and-forecast:api:brilon:forecastToday#time-stamp
	at org.eclipse.smarthome.core.thing.util.ThingHelper.ensureUniqueChannels(ThingHelper.java:159) ~[?:?]
	at org.eclipse.smarthome.core.thing.util.ThingHelper.ensureUniqueChannels(ThingHelper.java:151) ~[?:?]
	at org.eclipse.smarthome.core.thing.util.ThingHelper.ensureUniqueChannels(ThingHelper.java:147) ~[?:?]
	at org.eclipse.smarthome.core.thing.binding.builder.ThingBuilder.withChannel(ThingBuilder.java:74) ~[?:?]
	at org.eclipse.smarthome.binding.openweathermap.internal.handler.OpenWeatherMapWeatherAndForecastHandler.initialize(OpenWeatherMapWeatherAndForecastHandler.java:153) ~[?:?]
	at org.eclipse.smarthome.core.thing.binding.BaseThingHandler.thingUpdated(BaseThingHandler.java:208) ~[?:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:?]
	at org.eclipse.smarthome.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:153) [102:org.eclipse.smarthome.core:0.10.0.oh240]
	at org.eclipse.smarthome.core.internal.common.Invocation.call(Invocation.java:53) [102:org.eclipse.smarthome.core:0.10.0.oh240]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]
	at java.lang.Thread.run(Thread.java:748) [?:?]
2018-12-26 21:26:01.034 [DEBUG] [nWeatherMapWeatherAndForecastHandler] - Update weather and forecast data of thing 'openweathermap:weather-and-forecast:api:brilon'.
2018-12-26 21:26:01.044 [TRACE] [.connection.OpenWeatherMapConnection] - OpenWeatherMap request: URL = 'https://api.openweathermap.org/data/2.5/weather?appid=*****&lon=8.57151833297392&units=metric&lang=de&lat=51.394574451814094'
2018-12-26 21:26:01.517 [TRACE] [.connection.OpenWeatherMapConnection] - OpenWeatherMap response: status = 200, content = '{"coord":{"lon":8.57,"lat":51.39},"weather":[{"id":741,"main":"Fog","description":"Nebel","icon":"50n"}],"base":"stations","main":{"temp":2.19,"pressure":1029,"humidity":100,"temp_min":1,"temp_max":3},"visibility":650,"wind":{"speed":1.5,"deg":110},"clouds":{"all":90},"dt":1545853800,"sys":{"type":1,"id":1304,"message":0.0023,"country":"DE","sunrise":1545809429,"sunset":1545837776},"id":2944079,"name":"Brilon","cod":200}'
2018-12-26 21:26:01.538 [TRACE] [.connection.OpenWeatherMapConnection] - OpenWeatherMap request: URL = 'https://api.openweathermap.org/data/2.5/forecast?appid=*****&cnt=24&lon=8.57151833297392&units=metric&lang=de&lat=51.394574451814094'
2018-12-26 21:26:01.635 [TRACE] [.connection.OpenWeatherMapConnection] - OpenWeatherMap response: status = 200, content = '{"cod":"200","message":0.0023,"cnt":24,"list":[{"dt":1545858000,"main":{"temp":1.3,"temp_min":1.28,"temp_max":1.3,"pressure":1016.9,"sea_level":1044.14,"grnd_level":1016.9,"humidity":95,"temp_kf":0.03},"weather":[{"id":500,"main":"Rain","description":"Leichter Regen","icon":"10n"}],"clouds":{"all":64},"wind":{"speed":1.32,"deg":191.501},"rain":{"3h":0.01},"sys":{"pod":"n"},"dt_txt":"2018-12-26 21:00:00"},{"dt":1545868800,"main":{"temp":0.24,"temp_min":0.22,"temp_max":0.24,"pressure":1015.65,"sea_level":1043.04,"grnd_level":1015.65,"humidity":91,"temp_kf":0.02},"weather":[{"id":803,"main":"Clouds","description":"Überwiegend bewölkt","icon":"04n"}],"clouds":{"all":68},"wind":{"speed":1.32,"deg":156},"rain":{},"sys":{"pod":"n"},"dt_txt":"2018-12-27 00:00:00"},{"dt":1545879600,"main":{"temp":-1.35,"temp_min":-1.37,"temp_max":-1.35,"pressure":1014.92,"sea_level":1042.34,"grnd_level":1014.92,"humidity":85,"temp_kf":0.01},"weather":[{"id":801,"main":"Clouds","description":"Ein paar Wolken","icon":"02n"}],"clouds":{"all":12},"wind":{"speed":1.31,"deg":179.004},"rain":{},"sys":{"pod":"n"},"dt_txt":"2018-12-27 03:00:00"},{"dt":1545890400,"main":{"temp":-3.16,"temp_min":-3.17,"temp_max":-3.16,"pressure":1014.01,"sea_level":1041.4,"grnd_level":1014.01,"humidity":84,"temp_kf":0.01},"weather":[{"id":801,"main":"Clouds","description":"Ein paar Wolken","icon":"02n"}],"clouds":{"all":20},"wind":{"speed":1.31,"deg":172.001},"rain":{},"sys":{"pod":"n"},"dt_txt":"2018-12-27 06:00:00"},{"dt":1545901200,"main":{"temp":-2,"temp_min":-2,"temp_max":-2,"pressure":1013.58,"sea_level":1040.95,"grnd_level":1013.58,"humidity":89,"temp_kf":0},"weather":[{"id":803,"main":"Clouds","description":"Überwiegend bewölkt","icon":"04d"}],"clouds":{"all":56},"wind":{"speed":1.31,"deg":162.505},"rain":{},"sys":{"pod":"d"},"dt_txt":"2018-12-27 09:00:00"},{"dt":1545912000,"main":{"temp":3.12,"temp_min":3.12,"temp_max":3.12,"pressure":1012.86,"sea_level":1040.02,"grnd_level":1012.86,"humidity":91,"temp_kf":0},"weather":[{"id":802,"main":"Clouds","description":"Überwiegend bewölkt","icon":"03d"}],"clouds":{"all":44},"wind":{"speed":1.93,"deg":185},"rain":{},"sys":{"pod":"d"},"dt_txt":"2018-12-27 12:00:00"},{"dt":1545922800,"main":{"temp":3.24,"temp_min":3.24,"temp_max":3.24,"pressure":1011.93,"sea_level":1039.07,"grnd_level":1011.93,"humidity":88,"temp_kf":0},"weather":[{"id":803,"main":"Clouds","description":"Überwiegend bewölkt","icon":"04d"}],"clouds":{"all":80},"wind":{"speed":1.38,"deg":168},"rain":{},"sys":{"pod":"d"},"dt_txt":"2018-12-27 15:00:00"},{"dt":1545933600,"main":{"temp":0.86,"temp_min":0.86,"temp_max":0.86,"pressure":1011.93,"sea_level":1039.08,"grnd_level":1011.93,"humidity":87,"temp_kf":0},"weather":[{"id":803,"main":"Clouds","description":"Überwiegend bewölkt","icon":"04n"}],"clouds":{"all":56},"wind":{"speed":1.29,"deg":219.505},"rain":{},"sys":{"pod":"n"},"dt_txt":"2018-12-27 18:00:00"},{"dt":1545944400,"main":{"temp":-0.74,"temp_min":-0.74,"temp_max":-0.74,"pressure":1011.6,"sea_level":1038.81,"grnd_level":1011.6,"humidity":87,"temp_kf":0},"weather":[{"id":803,"main":"Clouds","description":"Überwiegend bewölkt","icon":"04n"}],"clouds":{"all":68},"wind":{"speed":1.29,"deg":192.503},"rain":{},"sys":{"pod":"n"},"dt_txt":"2018-12-27 21:00:00"},{"dt":1545955200,"main":{"temp":-1.29,"temp_min":-1.29,"temp_max":-1.29,"pressure":1011.59,"sea_level":1038.84,"grnd_level":1011.59,"humidity":84,"temp_kf":0},"weather":[{"id":802,"main":"Clouds","description":"Überwiegend bewölkt","icon":"03n"}],"clouds":{"all":44},"wind":{"speed":1.61,"deg":221.501},"rain":{},"sys":{"pod":"n"},"dt_txt":"2018-12-28 00:00:00"},{"dt":1545966000,"main":{"temp":-2.7,"temp_min":-2.7,"temp_max":-2.7,"pressure":1011.81,"sea_level":1039.17,"grnd_level":1011.81,"humidity":84,"temp_kf":0},"weather":[{"id":800,"main":"Clear","description":"Klarer Himmel","icon":"01n"}],"clouds":{"all":0},"wind":{"speed":2.06,"deg":229.001},"rain":{},"sys":{"pod":"n"},"dt_txt":"2018-12-28 03:00:00"},{"dt":1545976800,"main":{"temp":-3.79,"temp_min":-3.79,"temp_max":-3.79,"pressure":1012.32,"sea_level":1039.75,"grnd_level":1012.32,"humidity":84,"temp_kf":0},"weather":[{"id":800,"main":"Clear","description":"Klarer Himmel","icon":"01n"}],"clouds":{"all":0},"wind":{"speed":2.18,"deg":237.013},"rain":{},"sys":{"pod":"n"},"dt_txt":"2018-12-28 06:00:00"},{"dt":1545987600,"main":{"temp":-2.41,"temp_min":-2.41,"temp_max":-2.41,"pressure":1013.56,"sea_level":1040.91,"grnd_level":1013.56,"humidity":89,"temp_kf":0},"weather":[{"id":802,"main":"Clouds","description":"Überwiegend bewölkt","icon":"03d"}],"clouds":{"all":36},"wind":{"speed":2.07,"deg":232.501},"rain":{},"sys":{"pod":"d"},"dt_txt":"2018-12-28 09:00:00"},{"dt":1545998400,"main":{"temp":1.96,"temp_min":1.96,"temp_max":1.96,"pressure":1014.43,"sea_level":1041.54,"grnd_level":1014.43,"humidity":90,"temp_kf":0},"weather":[{"id":500,"main":"Rain","description":"Leichter Regen","icon":"10d"}],"clouds":{"all":64},"wind":{"speed":2.41,"deg":254.501},"rain":{"3h":0.05},"sys":{"pod":"d"},"dt_txt":"2018-12-28 12:00:00"},{"dt":1546009200,"main":{"temp":2,"temp_min":2,"temp_max":2,"pressure":1015.26,"sea_level":1042.46,"grnd_level":1015.26,"humidity":90,"temp_kf":0},"weather":[{"id":500,"main":"Rain","description":"Leichter Regen","icon":"10d"}],"clouds":{"all":88},"wind":{"speed":1.87,"deg":261.501},"rain":{"3h":0.09},"sys":{"pod":"d"},"dt_txt":"2018-12-28 15:00:00"},{"dt":1546020000,"main":{"temp":0.61,"temp_min":0.61,"temp_max":0.61,"pressure":1016.55,"sea_level":1043.89,"grnd_level":1016.55,"humidity":89,"temp_kf":0},"weather":[{"id":500,"main":"Rain","description":"Leichter Regen","icon":"10n"}],"clouds":{"all":80},"wind":{"speed":2.47,"deg":240.503},"rain":{"3h":0.11},"sys":{"pod":"n"},"dt_txt":"2018-12-28 18:00:00"},{"dt":1546030800,"main":{"temp":0.21,"temp_min":0.21,"temp_max":0.21,"pressure":1017.82,"sea_level":1045.11,"grnd_level":1017.82,"humidity":92,"temp_kf":0},"weather":[{"id":500,"main":"Rain","description":"Leichter Regen","icon":"10n"}],"clouds":{"all":88},"wind":{"speed":2.76,"deg":243.507},"rain":{"3h":0.13},"sys":{"pod":"n"},"dt_txt":"2018-12-28 21:00:00"},{"dt":1546041600,"main":{"temp":0.4,"temp_min":0.4,"temp_max":0.4,"pressure":1018.57,"sea_level":1045.9,"grnd_level":1018.57,"humidity":93,"temp_kf":0},"weather":[{"id":500,"main":"Rain","description":"Leichter Regen","icon":"10n"}],"clouds":{"all":88},"wind":{"speed":2.96,"deg":240.004},"rain":{"3h":0.15},"sys":{"pod":"n"},"dt_txt":"2018-12-29 00:00:00"},{"dt":1546052400,"main":{"temp":0.35,"temp_min":0.35,"temp_max":0.35,"pressure":1018.45,"sea_level":1045.87,"grnd_level":1018.45,"humidity":96,"temp_kf":0},"weather":[{"id":500,"main":"Rain","description":"Leichter Regen","icon":"10n"}],"clouds":{"all":76},"wind":{"speed":3.17,"deg":224.5},"rain":{"3h":0.13},"sys":{"pod":"n"},"dt_txt":"2018-12-29 03:00:00"},{"dt":1546063200,"main":{"temp":0.5,"temp_min":0.5,"temp_max":0.5,"pressure":1018.63,"sea_level":1045.99,"grnd_level":1018.63,"humidity":95,"temp_kf":0},"weather":[{"id":500,"main":"Rain","description":"Leichter Regen","icon":"10n"}],"clouds":{"all":88},"wind":{"speed":3.47,"deg":232.5},"rain":{"3h":0.12},"sys":{"pod":"n"},"dt_txt":"2018-12-29 06:00:00"},{"dt":1546074000,"main":{"temp":1.3,"temp_min":1.3,"temp_max":1.3,"pressure":1019.07,"sea_level":1046.41,"grnd_level":1019.07,"humidity":97,"temp_kf":0},"weather":[{"id":500,"main":"Rain","description":"Leichter Regen","icon":"10d"}],"clouds":{"all":88},"wind":{"speed":4.37,"deg":231.003},"rain":{"3h":0.2},"sys":{"pod":"d"},"dt_txt":"2018-12-29 09:00:00"},{"dt":1546084800,"main":{"temp":2.61,"temp_min":2.61,"temp_max":2.61,"pressure":1018.42,"sea_level":1045.65,"grnd_level":1018.42,"humidity":97,"temp_kf":0},"weather":[{"id":500,"main":"Rain","description":"Leichter Regen","icon":"10d"}],"clouds":{"all":92},"wind":{"speed":5.11,"deg":235},"rain":{"3h":0.19},"sys":{"pod":"d"},"dt_txt":"2018-12-29 12:00:00"},{"dt":1546095600,"main":{"temp":2.6,"temp_min":2.6,"temp_max":2.6,"pressure":1016.54,"sea_level":1043.62,"grnd_level":1016.54,"humidity":95,"temp_kf":0},"weather":[{"id":500,"main":"Rain","description":"Leichter Regen","icon":"10d"}],"clouds":{"all":68},"wind":{"speed":6.17,"deg":230.5},"rain":{"3h":0.13},"sys":{"pod":"d"},"dt_txt":"2018-12-29 15:00:00"},{"dt":1546106400,"main":{"temp":2.61,"temp_min":2.61,"temp_max":2.61,"pressure":1014.79,"sea_level":1041.94,"grnd_level":1014.79,"humidity":95,"temp_kf":0},"weather":[{"id":500,"main":"Rain","description":"Leichter Regen","icon":"10n"}],"clouds":{"all":92},"wind":{"speed":7.96,"deg":236.505},"rain":{"3h":0.1},"sys":{"pod":"n"},"dt_txt":"2018-12-29 18:00:00"}],"city":{"id":2944079,"name":"Brilon","coord":{"lat":51.4,"lon":8.5833},"country":"DE","population":27030}}'
2018-12-26 21:26:01.674 [TRACE] [.connection.OpenWeatherMapConnection] - OpenWeatherMap request: URL = 'https://api.openweathermap.org/data/2.5/forecast/daily?appid=*****&cnt=6&lon=8.57151833297392&units=metric&lang=de&lat=51.394574451814094'
2018-12-26 21:26:01.742 [TRACE] [.connection.OpenWeatherMapConnection] - Exception occurred during execution: org.eclipse.jetty.client.HttpResponseException: HTTP protocol violation: Authentication challenge without WWW-Authenticate header
java.util.concurrent.ExecutionException: org.eclipse.jetty.client.HttpResponseException: HTTP protocol violation: Authentication challenge without WWW-Authenticate header
	at org.eclipse.jetty.client.util.FutureResponseListener.getResult(FutureResponseListener.java:118) ~[?:?]
	at org.eclipse.jetty.client.util.FutureResponseListener.get(FutureResponseListener.java:101) ~[?:?]
	at org.eclipse.jetty.client.HttpRequest.send(HttpRequest.java:684) ~[?:?]
	at org.eclipse.smarthome.binding.openweathermap.internal.connection.OpenWeatherMapConnection.getResponse(OpenWeatherMapConnection.java:244) ~[?:?]
	at org.eclipse.smarthome.binding.openweathermap.internal.connection.OpenWeatherMapConnection.lambda$1(OpenWeatherMapConnection.java:235) ~[?:?]
	at org.eclipse.smarthome.core.cache.ExpiringCache.refreshValue(ExpiringCache.java:97) ~[?:?]
	at org.eclipse.smarthome.core.cache.ExpiringCache.getValue(ExpiringCache.java:68) ~[?:?]
	at org.eclipse.smarthome.core.cache.ExpiringCacheMap.get(ExpiringCacheMap.java:198) ~[?:?]
	at org.eclipse.smarthome.core.cache.ExpiringCacheMap.putIfAbsentAndGet(ExpiringCacheMap.java:127) ~[?:?]
	at org.eclipse.smarthome.core.cache.ExpiringCacheMap.putIfAbsentAndGet(ExpiringCacheMap.java:112) ~[?:?]
	at org.eclipse.smarthome.binding.openweathermap.internal.connection.OpenWeatherMapConnection.getResponseFromCache(OpenWeatherMapConnection.java:235) ~[?:?]
	at org.eclipse.smarthome.binding.openweathermap.internal.connection.OpenWeatherMapConnection.getDailyForecastData(OpenWeatherMapConnection.java:161) ~[?:?]
	at org.eclipse.smarthome.binding.openweathermap.internal.handler.OpenWeatherMapWeatherAndForecastHandler.requestData(OpenWeatherMapWeatherAndForecastHandler.java:170) ~[?:?]
	at org.eclipse.smarthome.binding.openweathermap.internal.handler.AbstractOpenWeatherMapHandler.updateData(AbstractOpenWeatherMapHandler.java:133) ~[?:?]
	at org.eclipse.smarthome.binding.openweathermap.internal.handler.OpenWeatherMapAPIHandler.updateThing(OpenWeatherMapAPIHandler.java:176) ~[?:?]
	at org.eclipse.smarthome.binding.openweathermap.internal.handler.OpenWeatherMapAPIHandler.updateThings(OpenWeatherMapAPIHandler.java:167) ~[?:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:?]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) [?:?]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) [?:?]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]
	at java.lang.Thread.run(Thread.java:748) [?:?]
Caused by: org.eclipse.jetty.client.HttpResponseException: HTTP protocol violation: Authentication challenge without WWW-Authenticate header
	at org.eclipse.jetty.client.AuthenticationProtocolHandler$AuthenticationListener.onComplete(AuthenticationProtocolHandler.java:197) ~[?:?]
	at org.eclipse.jetty.client.ResponseNotifier.notifyComplete(ResponseNotifier.java:216) ~[?:?]
	at org.eclipse.jetty.client.ResponseNotifier.notifyComplete(ResponseNotifier.java:208) ~[?:?]
	at org.eclipse.jetty.client.HttpReceiver.terminateResponse(HttpReceiver.java:470) ~[?:?]
	at org.eclipse.jetty.client.HttpReceiver.responseSuccess(HttpReceiver.java:416) ~[?:?]
	at org.eclipse.jetty.client.http.HttpReceiverOverHTTP.messageComplete(HttpReceiverOverHTTP.java:316) ~[?:?]
	at org.eclipse.jetty.http.HttpParser.handleContentMessage(HttpParser.java:605) ~[?:?]
	at org.eclipse.jetty.http.HttpParser.parseContent(HttpParser.java:1675) ~[?:?]
	at org.eclipse.jetty.http.HttpParser.parseNext(HttpParser.java:1523) ~[?:?]
	at org.eclipse.jetty.client.http.HttpReceiverOverHTTP.parse(HttpReceiverOverHTTP.java:172) ~[?:?]
	at org.eclipse.jetty.client.http.HttpReceiverOverHTTP.process(HttpReceiverOverHTTP.java:135) ~[?:?]
	at org.eclipse.jetty.client.http.HttpReceiverOverHTTP.receive(HttpReceiverOverHTTP.java:73) ~[?:?]
	at org.eclipse.jetty.client.http.HttpChannelOverHTTP.receive(HttpChannelOverHTTP.java:133) ~[?:?]
	at org.eclipse.jetty.client.http.HttpConnectionOverHTTP.onFillable(HttpConnectionOverHTTP.java:155) ~[?:?]
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:281) ~[?:?]
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:102) ~[?:?]
	at org.eclipse.jetty.io.ssl.SslConnection.onFillable(SslConnection.java:291) ~[?:?]
	at org.eclipse.jetty.io.ssl.SslConnection$3.succeeded(SslConnection.java:151) ~[?:?]
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:102) ~[?:?]
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:118) ~[?:?]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:333) ~[?:?]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:310) ~[?:?]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:168) ~[?:?]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:126) ~[?:?]
	at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:366) ~[?:?]
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:762) ~[?:?]
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:680) ~[?:?]
	... 1 more
2018-12-26 21:26:01.859 [DEBUG] [.connection.OpenWeatherMapConnection] - OpenWeatherMap server responded with status code 401: Invalid API key.
2018-12-26 21:26:01.864 [WARN ] [nWeatherMapWeatherAndForecastHandler] - @text/offline.conf-error-invalid-apikey
2018-12-26 21:26:01.918 [DEBUG] [nWeatherMapWeatherAndForecastHandler] - Removing daily forecast channel groups.
2018-12-26 21:26:02.024 [DEBUG] [nWeatherMapWeatherAndForecastHandler] - Update channel 'id' of group 'station' with new state '2944079'.
```
BTW: is there a key missing in the language definition:
```
2018-12-26 21:26:01.864 [WARN ] [nWeatherMapWeatherAndForecastHandler] - @text/offline.conf-error-invalid-apikey
```

--- a/extensions/binding/org.eclipse.smarthome.binding.openweathermap/src/main/java/org/eclipse/smarthome/binding/openweathermap/internal/handler/AbstractOpenWeatherMapHandler.java
+++ b/extensions/binding/org.eclipse.smarthome.binding.openweathermap/src/main/java/org/eclipse/smarthome/binding/openweathermap/internal/handler/AbstractOpenWeatherMapHandler.java
@@ -208,7 +208,14 @@ public abstract class AbstractOpenWeatherMapHandler extends BaseThingHandler {
         if (callback != null) {
             for (ChannelBuilder channelBuilder : callback.createChannelBuilders(
                     new ChannelGroupUID(getThing().getUID(), channelGroupId), channelGroupTypeUID)) {
-                channels.add(channelBuilder.build());
+                Channel newChannel = channelBuilder.build(),
+                        existingChannel = getThing().getChannel(newChannel.getUID().getId());
+                if (existingChannel != null) {
+                    logger.trace("Thing '{}' already has an existing channel '{}'. Omit adding new channel '{}'.",
+                            getThing().getUID(), existingChannel.getUID(), newChannel.getUID());
+                    continue;
+                }
+                channels.add(newChannel);
             }
         }
         return channels;

--- a/extensions/binding/org.eclipse.smarthome.binding.openweathermap/src/main/java/org/eclipse/smarthome/binding/openweathermap/internal/handler/OpenWeatherMapWeatherAndForecastHandler.java
+++ b/extensions/binding/org.eclipse.smarthome.binding.openweathermap/src/main/java/org/eclipse/smarthome/binding/openweathermap/internal/handler/OpenWeatherMapWeatherAndForecastHandler.java
@@ -170,7 +170,6 @@ public class OpenWeatherMapWeatherAndForecastHandler extends AbstractOpenWeather
                     dailyForecastData = connection.getDailyForecastData(location, forecastDays);
                 } catch (OpenWeatherMapConfigurationException e) {
                     if (e.getCause() instanceof HttpResponseException) {
-                        logger.warn(e.getLocalizedMessage());
                         forecastDays = 0;
                         Configuration editConfig = editConfiguration();
                         editConfig.put(CONFIG_FORECAST_DAYS, 0);

